analysis/import/CSVImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/sourcemonitor/AttributeDescriptors.kt

```diff
@@ -10,7 +10,7 @@ fun getAttributeDescriptors(): Map<String, AttributeDescriptor> {
     descriptors["classes"] = AttributeDescriptor(title = "Number of Classes", description = "Number of classes", link = ghLink)
     descriptors["functions_per_class"] = AttributeDescriptor(title = "Functions per Class", description = "Number of functions per class", link = ghLink)
     descriptors["average_statements_per_function"] = AttributeDescriptor(title = "Average Statements per Function", description = "Average number of statements per method", link = ghLink)
-    descriptors["max_function_mcc"] = AttributeDescriptor(title = "Maximum Cyclic Complexity", description = "Maximum cyclic complexity of a function", link = ghLink)
+    descriptors["max_function_mcc"] = AttributeDescriptor(title = "Maximum Cyclic Complexity", description = "Maximum cyclic complexity based on paths through a function by McCabe", link = ghLink)
```

Comment:
In metricTitles.ts (old: metricNames.ts) the title for mcc ist "Cyclomatic Complexity". Either change it here or there to avoid inconsistency.

Reply:
@phanlezz please remember that we can also have a completely independent file that has been generated by something else than our own importers/parsers. Thus, we probably have to keep the fallback mechanism in place as we can not determine what they define.

Reply:
@BridgeAR Yes, the fallback should stay ofcourse. My point is, that we don't need to update the definitions in the .ts or add more, since we added the attributeDescriptors to have a more flexible solution.